### PR TITLE
fix: State Manager component, and TextArea component rendering issues

### PR DESCRIPTION
**Pull request by Canuckaholic**
## Description

Fixes https://github.com/c6o/node-monorepo/issues/965

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: packages/web/src/formInputs/textArea.ts [hunk 1]
```diff
@@ -94,18 +96,22 @@ export class TextArea extends mix(TextAreaElement).with(EntityStoreMixin, Entity
...
 
-        if (this.markdown) {
-            const easyMDE = new EasyMDE({
-                element: this.root.querySelector('textarea'),
-                autoDownloadFontAwesome: false,
-                minHeight: this.minHeight,
-                showIcons: ['code', 'table', 'horizontal-rule']
-            })
+        this.textAreaDisposer = observe(this.store, 'entity', () => {
+            if (this.markdown) {
```
Comment (nsainaney) [inline packages/web/src/formInputs/textArea.ts:100]:
```suggestion
            if (this.markdown && this.entity) {
```

Comment (nsainaney) [reply packages/web/src/formInputs/textArea.ts:100]: Should you clear the markdown if the entity became null?

**Review by nsainaney — CHANGES_REQUESTED** [review]

File: packages/web/src/store/state.ts [hunk 2]
```diff
@@ -67,20 +67,20 @@ export class StoreState extends MobxLitElement {
...
-                <c6o-errors heading="The system encountered an error:" .store=${this.store}></c6o-errors>
             `
         } else {
             return this.container ? html`
                 <div c6o="container">
                     <c6o-loading loading-message=${this.loadingMessage} ?simple=${this.simpleUI}></c6o-loading>
                 </div>
-            ` : html`<c6o-loading loading-message=${this.loadingMessage} ?simple=${this.simpleUI}></c6o-loading>`
+            ` : html`
+                <c6o-loading loading-message=${this.loadingMessage} ?simple=${this.simpleUI}></c6o-loading>
```
Comment (nsainaney) [inline packages/web/src/store/state.ts:82]: There should be a default value of `loadingMessage`. Something generic like `Loading...`

**Review by nsainaney — CHANGES_REQUESTED** [review]

File: packages/web/src/store/state.ts (hunk 2 above)
Comment (Canuckaholic) [reply packages/web/src/store/state.ts:82]: There is... see loading.ts

```
@property({ type: String, attribute: 'loading-message' })
    loadingMessage = 'Loading...'````

Comment (nsainaney) [reply packages/web/src/store/state.ts:82]: Not sure if you are supposed to initialize it in the decorator or inline

**Review by nsainaney — APPROVED** [review]

**Pull request merged**